official/NormeDiProgetto/res/sections/supportProcesses.tex

```diff
@@ -12,6 +12,43 @@ \subsubsection{Risultati attesi dal processo}
 \item la corretta stesura dei documenti segnalati;
 \item l'individuazione degli standard a cui tali documenti devono aderire.
 \end{itemize}
+
+\subsubsection{Procedure automatiche a supporto}
+Nello sviluppo della documentazione vengono utilizzate delle procedure automatiche per supportare la generazione di interi documenti o di alcune loro parti. Di seguito verranno descritte le procedure automatiche per la generazione dell'elenco di tabelle e figure e per la generazione del glossario.
+\paragraph{Generazione della lista delle tabelle e della lista delle figure} \mbox{} \\
+Le tabelle e le figure inserite nel documento vengono automaticamente indicizzate durante la compilazione del codice \LaTeX{} attraverso l'uso dei comandi \textit{listoftables} e \textit{listoffigures}. Questi due comandi si basano sull'associazione della tabella, o figura, ad una didascalia (\textit{caption}). Grazie a questa associazione viene mostrato, subito dopo l'indice del documento, la lista delle tabelle e/o delle figure. È compito di ogni membro del team inserire tale \textit{caption} durante la stesura di una tabella o l'inserimento di una figura.
+
+\paragraph{Generazione del glossario} \mbox{} \\
+Il glossario dei termini a supporto dei documenti è interamente generato con l'utilizzo di uno script Perl (\textbf{P}ractical \textbf{E}xtraction and \textbf{R}eport \textbf{L}anguage) sviluppato internamente. Un file XML (e\textbf{X}tensible \textbf{M}arkup \textbf{L}anguage) contente le definizioni dei termini in ordine casuale. Ogni voce del glossario corrisponde al seguente tag:
+\begin{verbatim}
+<term>
+    <word>Termine</word>
+    <extended>Opzionale, descrizione estesa dell'acronimo.</extended>
+    <plural>Opzionale, plurale del termine.</plural>
+    <definition>Definizione.</definition>
+</term>
+\end{verbatim}
+regolamentato dalla seguente istanza di XML Schema:
+\begin{verbatim}
+<xs:element name="glossary">
+    <xs:complexType>
+        <xs:sequence>
+            <xs:element name="term" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="word" type="xs:ID" />
+                        <xs:element name="plural" type="xs:string" minOccurs="0" />
+                        <xs:element name="extended" type="xs:string" minOccurs="0" />
+                        <xs:element name="definition" type="xs:string" maxOccurs="unbounded" />
```

Comment:
@mdipirro In questa riga nel pdf va fuori dal foglio! Non c'e' modo di farla andare a capo? :blush: